build.gradle

```diff
@@ -5,8 +5,8 @@ allprojects {
     compileJava.options.encoding = 'UTF-8'
     compileTestJava.options.encoding = 'UTF-8'
 
-    sourceCompatibility = 1.8
-    targetCompatibility = 1.8
+    sourceCompatibility = 9
+    targetCompatibility = 9
```

Comment:
You should have precised that your changes require Java 9!

Reply:
~~precised~~ specified

Reply:
Sorry about that. I updated the description.

Reply:
I didn't use Java 11 methods in this PR, but I guess Java 11 could be used instead.